### ISSUE TITLE
Path to markitdown

### ISSUE THREAD

**Issue by soccken** [microsoft/markitdown#204]
I installed it by running [pip install markitdown] in Windows PowerShell.

After that, I ran [markitdown path-to-file.pdf -o document.md], but I got a problem where "markitdown" was not recognized as an operable program.
Is this a problem with the Path?

**Comment by l-lumin** [comment]
hi, can you share your Path environment variable? I think you should run it in venv or use pipx, rather than installing it directly with pip

**Comment by mrbarua** [comment]
Even I have the same issue "markitdown command not found"

**Comment by boredchilada** [comment]
You can use my webgui if you'd like while you get this fixed. We do not save any information.

we also added batch mode.

mitdown.ca 

![Image](https://github.com/user-attachments/assets/8e95bc2d-e426-4c37-ae5e-160fe5c6db2b)

**Comment by mrbarua** [comment]
Hello @soccken 
I solved it after reinstalling Python.

**Comment by Walterfraser** [comment]
> I installed it by running [pip install markitdown] in Windows PowerShell.
> 
> After that, I ran [markitdown path-to-file.pdf -o document.md], but I got a problem where "markitdown" was not recognized as an operable program. Is this a problem with the Path?

It installs it into your python script directory, for me this was:

C:\Users\Username\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\Scripts\

You can install a virtual environment in a folder by typing:

python3 -m venv venv

Then use the following to activate it: venv\scripts\activate

This is relative to Windows, colleagues on mac had no issue with this

**Comment by tiansiyuan** [comment]
Same on Ubuntu 20.04.

**Comment by KiryUp** [comment]
Can anyone please explain, or at least point to a documentation that shows how to make this work globally ? i tried this on ubuntu too and i get the same message "markitdown command not found"". From the doc it simply says that after a "pip install" it should be possible to call it from a cmd line interface just like a tool, but it's not the case. Thanks for the help

**Comment by KiryUp** [comment]
just a follow up on how I fixed the issue on my hand. 
I was using an old version of python (3.8 on ubuntu WSL), which seemed to install an "incomplete" vresion folder inside ~/.local/lib/python3.8/site-packages that's why I was confused.  
Eventually I used "pyenv" to install a new version and installed a 3.12 version and made it my "global" version, then did the install again via "pip install markitdown" and it worked.
Hope this help !

**Comment by tiansiyuan** [comment]
Yes, it does not work with Python 3.8 (the default Python3 version on Ubuntu 2004.

It works with 3.12. 

I tried with a 3.12 docker image, python:3.12.

Thanks

**Comment by pchuri** [comment]
I also experienced the "markitdown command not found" issue and solved it with these steps:

1. I ran `pip show markitdown` to find the actual installation path.
2. In my case, it was installed at `C:\Users\USER\AppData\Roaming\Python\Python312\Scripts`.
3. I added this path to the Windows PATH environment variable (Control Panel > System > Advanced System Settings > Environment Variables > Edit Path).
4. After opening a new command prompt, the markitdown command worked correctly.

The script installation location may vary depending on your Python installation method and environment, so it's important to check the exact path with `pip show markitdown` and then add that to your PATH.

**Comment by AndreiCherniaev** [comment]
Ubuntu 25
Install
```
pip install 'markitdown[pdf, docx, pptx]' --break-system-packages
```
Append to $PATH variable
```
sudo sh -c 'cat << EOF > /etc/profile.d/my-path-additions.sh
export PATH="\$PATH:$HOME/.local/bin"
EOF'
```
Logout, login (or just reboot). Then you can use `markitdown --help`

**Comment by Shreyashmohadikar** [comment]
I also encountered the “**markitdown command not found**” issue and resolved it as follows:

After creating and activating the virtual environment, use `python -m pip` instead of just `pip`. Using only `pip install package` may install the package in the globally installed Python environment instead of the activated virtual environment.

Or alternatively you can use `pip install uv` and then `uv pip install [package]`
